generator.json

```diff
@@ -77,6 +77,44 @@
                     "glPointSizePointerOES"
                 ]
             },
+            "glIntAsPtr": {
+                "EdgeFlagPointer": ["pointer"],
+                "EdgeFlagPointerEXT": ["pointer"],
+                "EdgeFlagPointerListIBM": ["pointer"],
+                "VertexPointer": ["pointer"],
+                "VertexPointerEXT": ["pointer"],
+                "VertexPointerListIBM": ["pointer"],
+                "VertexPointervINTEL": ["pointer"],
+                "ColorPointer": ["pointer"],
+                "ColorPointerEXT": ["pointer"],
+                "ColorPointerListIBM": ["pointer"],
+                "ColorPointervINTEL": ["pointer"],
+                "IndexPointer": ["pointer"],
+                "IndexPointerEXT": ["pointer"],
+                "IndexPointerListIBM": ["pointer"],
+                "NormalPointer": ["pointer"],
+                "NormalPointerEXT": ["pointer"],
+                "NormalPointerListIBM": ["pointer"],
+                "NormalPointervINTEL": ["pointer"],
+                "TexCoordPointer": ["pointer"],
+                "TexCoordPointerEXT": ["pointer"],
+                "TexCoordPointerListIBM": ["pointer"],
+                "TexCoordPointervINTEL": ["pointer"],
+                "MultiTexCoordPointerEXT": ["pointer"],
+                "FogCoordPointer": ["pointer"],
+                "FogCoordPointerEXT": ["pointer"],
+                "FogCoordPointerListIBM": ["pointer"],
+                "SecondaryColorPointer": ["pointer"],
+                "SecondaryColorPointerEXT": ["pointer"],
+                "SecondaryColorPointerListIBM": ["pointer"],
+                "VertexAttribPointer": ["pointer"],
+                "VertexAttribPointerARB": ["pointer"],
+                "VertexAttribPointerNV": ["pointer"],
+                "VertexAttribIPointer": ["pointer"],
+                "VertexAttribIPointerEXT": ["pointer"],
+                "TangentPointerEXT": ["pointer"],
+                "BinormalPointerEXT": ["pointer"]
+            },
```

Comment:
Add to the end of Section 2.10 (Vertex Array Objects):

    "The following commands:

        void VertexArrayVertexOffsetEXT(uint vaobj, uint buffer,
                                        int size, enum type, sizei stride,
                                        intptr offset);
        void VertexArrayColorOffsetEXT(uint vaobj, uint buffer,
                                       int size, enum type, sizei stride,
                                       intptr offset);
        void VertexArrayEdgeFlagOffsetEXT(uint vaobj, uint buffer,
                                          sizei stride, intptr offset);
        void VertexArrayIndexOffsetEXT(uint vaobj, uint buffer,
                                       enum type, sizei stride,
                                       intptr offset);
        void VertexArrayNormalOffsetEXT(uint vaobj, uint buffer,
                                        enum type, sizei stride,
                                        intptr offset);
        void VertexArrayTexCoordOffsetEXT(uint vaobj, uint buffer,
                                          int size, enum type, sizei stride,
                                          intptr offset);
        void VertexArrayMultiTexCoordOffsetEXT(uint vaobj, uint buffer,
                                               enum texunit,
                                               int size, enum type, sizei stride,
                                               intptr offset);
        void VertexArrayFogCoordOffsetEXT(uint vaobj, uint buffer,
                                          enum type, sizei stride,
                                          intptr offset);
        void VertexArraySecondaryColorOffsetEXT(uint vaobj, uint buffer,
                                                int size, enum type,
                                                sizei stride, intptr offset);
        void VertexArrayVertexAttribOffsetEXT(uint vaobj, uint buffer,
                                              uint index, int size,
                                              enum type, boolean normalized,
                                              sizei stride, intptr offset);
        void VertexArrayVertexAttribIOffsetEXT(uint vaobj, uint buffer,
                                               uint index, int size,
                                               enum type, sizei stride,
                                               intptr offset);

    operate identically to the corresponding command where "VertexArray"
    is deleted from the name and "Offset" is changed to "Pointer" (and
    extension suffixes are dropped or updated appropriately) except,
    1) rather than updating the currently in use vertex array object,
    these "VertexArray" commands update the vertex array object named
    by the initial vaobj parameter, and 2) the corresponding command
    operates as if the ARRAY_BUFFER buffer binding is bound to the
    buffer named by the buffer parameter.  The remaining parameters
    following the initial vaobj parameter for the listed "VertexArray"
    commands match the parameters for the corresponding non-"VertexArray"
    vertex array specification command and are interpreted as they
    are for the non-"VertexArray" vertex array specification command.
    The one exception is that final intptr offset parameter of the
    "VertexArray" command is cast to the final pointer parameter of the
    non-"VertexArray" command (intptr is guaranteed to be large enough
    to store any pointer value).